pkg/controller/plan/kubevirt.go

```diff
@@ -980,22 +980,22 @@ func (r *KubeVirt) UpdateVmByConvertedConfig(vm *plan.VMStatus, pod *core.Pod, s
 
 	switch r.Source.Provider.Type() {
 	case api.Ova:
-		vmConfigBytes, err := io.ReadAll(resp.Body)
+		vmConf, err := io.ReadAll(resp.Body)
 		if err != nil {
-			return liberr.Wrap(err)
+			return err
```

Comment:
In other areas of the file, we’re using liberr.Wrap, which provides additional context and tracing. Given that I’m still getting familiar with the codebase, having the error trace could be really helpful for debugging and understanding the flow of issues. Would it make sense to wrap the error here as well for consistency and better traceability?

Reply:
I was thinking to move the `liberr.warp` out of this function to make apply it on all errors, but would be too much changes and did not return this back.

Reply:
fixed